### PR TITLE
Adding vagrant script

### PR DESCRIPTION
**Pull request by duleorlovic**
Added Vagrant script.
Later we can use this to deploy to DigitalOcean.
I also switch to rails default config/secrets.yml file and put my twitter keys so new users can easily test application in their virtual machine.

I see that seeds.rb is not idempotent (but it should be) but this is another task.

### CODE REVIEW

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -8,10 +8,10 @@
     config.storage = :aws
     config.aws_acl = :public_read
     config.aws_credentials = {
-      access_key_id:     ENV.fetch('AWS_ACCESS_KEY_ID'),
-      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
+      access_key_id:     aws_access_key_id,
```
Comment (wasi) [inline config/initializers/carrierwave.rb]: Doesn't that have to be `Rails.application.secrets.aws_access_key_id`?

File: config/secrets.yml [hunk 2]
```diff
@@ -13,10 +13,32 @@
...
 
+  twitter_app_id: N01qhL9a5erk5Zjga0GIi9I2K
+  twitter_app_secret: fZSN4NHJIzkKXLkz88sxi9z6EwIVyDWxiRhgqd03bKBFLr4pVx
+
+  facebook_app_id: <%= ENV['FACEBOOK_APP_ID'] %>
+  facebook_app_secret: <%= ENV['FACEBOOK_APP_SECRET'] %>
+
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
```
Comment (wasi) [inline config/secrets.yml]: s3 credentials are missing in production

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  # end
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  # config.vm.provision "shell", inline <<-SHELL
+  #   sudo apt-get install apache2
+  # SHELL
+
+  config.vm.provision "shell", inline: <<-SHELL
```
Comment (wasi) [inline Vagrantfile]: actually I hoped to provision with docker. that way we could potentially use docker for deployment aswell

File: config/initializers/carrierwave.rb (hunk 1 above)
Comment (duleorlovic) [inline config/initializers/carrierwave.rb]: Yes. My bad commit :(

File: Vagrantfile (hunk 3 above)
Comment (duleorlovic) [inline Vagrantfile]: I do not have experience with docker tool.
If this is accepted, I think we should update README with instructions.

File: config/secrets.yml (hunk 2 above)
Comment (duleorlovic) [inline config/secrets.yml]: Good catch.
To simplify this yml file, I think we could use

test:
   << *default
production:
  << *default
as it is in database.yml

File: Vagrantfile (hunk 3 above)
Comment (wasi) [inline Vagrantfile]: @duleorlovic would you agree to split the bounty into two (the vagrant and docker part)? you can also start on the docker thing, its pretty straight forward and I can give you some advise (even though I do not have so much experience with docker aswell).
that would adress the 2 main parts: having vagrant up very soon and working on docker aswell

Comment (duleorlovic) [inline Vagrantfile]: Sure. Please send me more info. I'm interested.

**Comment by duleorlovic** [comment]
Should I rebase and create new pull request ?

**Comment by wasi** [comment]
I merged the Vagrant file.

**Pull request closed without merging**